src/App.php

```diff
@@ -132,6 +132,16 @@ class App
 	 */
 	private $process;
 
+	/**
+	 * Returns the arguments of the query
+	 *
+	 * @return Arguments
+	 */
+	public function getArgs()
```

Comment:
We don't need this access if we use `BaseObject::getClass(App\Arguments::class)`